modules/ding_user/ding_user.module

```diff
@@ -386,13 +387,8 @@ function ding_user_form_user_admin_settings_alter(&$form, &$form_state, $form_id
   );
 
   $form['ding_user']['ding_user_pincode_length'] = array(
-    '#type' => 'radios',
+    '#type' => 'textfield',
```

Comment:
These are changes to the general `ding_user` module but not in the SAL branch. That does not seem right.

If we do not want to make the change generic then we might as well keep all changes in the `aleph` module through form alters.